tagstudio/src/core/media_types.py

```diff
@@ -307,6 +327,12 @@ class MediaCategories:
         ".xls",
         ".xlsx",
     }
+    _SCRIPT_SET: set[str] = {
```

Comment:
You currently aren't registering script files as a media category, see below. Once added as a category, you'd also need to update thumb_renderer.

Reply:
You could practically handle it as a text file

Reply:
I realise there is a text type. so sorry.